README.md

```diff
@@ -1,5 +1,14 @@
 # Redis Labs helm repository
 
+## GKE quickstart
+
+```bash
+password=$(gcloud container clusters describe YOUR_SETUP_NAME --zone us-central1-a | grep password | cut -d":" -f 2 | tr -d " ")
```

Comment:
" us-central1-a" can be changed to ZONE to be more generic

Reply:
done